pnpm-lock.yaml

```diff
@@ -455,6 +455,7 @@ importers:
       unist-util-visit: ^4.1.0
       vfile: ^5.3.2
       vite: ~3.1.3
+      vitefu: ^0.0.1
```

Comment:
i see that this is your package, what do you think about bumping this to 0.1.0? I'd hate for us to have to upgrade every time you fix a bug there.

Reply:
Yeah I'm planning to do so quite soon once I get the API ironed out with Dominik from `vite-plugin-svelte`. And making sure it really works.

I can cut 0.1.0 once this CI passes, along with https://github.com/sveltejs/vite-plugin-svelte/pull/478 and an upcoming PR to vite-plugin-solid

Reply:
Released and updated to 0.1.0 👍